.github/actions/setup-rust/tests/test_copy_stdlib.py

```
@@ -0,0 +1,37 @@
+"""Tests for copy_openbsd_stdlib.py script."""
+
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+
+def run_script(script: Path, *args: str) -> subprocess.CompletedProcess[str]:
```

Ruff (0.12.2)

Missing docstring in public function

(D103)